### DISCUSSION TITLE
Welcome to Ozon3 Discussions!

### DISCUSSION THREAD

**Discussion by Milind220** [Ozon3Org/Ozon3#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of the Ozon3 community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Let us know if you want to contribute but need help!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Milind220** [comment]
I'm here to help guys! I love open-source and I'd for anyone and everyone to get involved.

Let me know if I can do anything to help you get started :)
🙌🏽🥳

**Reply by Milind220** [reply]
Remember to read the [CONTRIBUTING.md](https://github.com/Milind220/Ozone/blob/dev/CONTRIBUTING.md) before contributing!

**Reply by ubongab** [reply]
Hi Milind220,

I'd love to contribute to this project!

**Reply by Milind220** [reply]
Hey @ubongab that's great! I see you've already found a section to help with :) 

Good luck with issue #17!

**Comment by iGotAdigBick** [comment]
Hi @Milind220 Is this an active community? I want to begin with open source projects in a fun community

**Reply by Milind220** [reply]
@iGotAdigBick Yes! We're a small but growing community of active contributors! New features are being added to Ozone every day!

**Comment by Miller2014** [comment]
Hi Milind220, 

I am keen to contribute and join this community.  I am relatively new to contributing to open source projects.  I have read the CONTRIBUTING.md which really helped.  I have forked the repository and have it set up in VS Code.  Before I start on some of the beginners issues, i thought I would run the units test to check i have a clean starting point.  I had a couple of failures which I would like to ask about, it seems to be a "vcr.errors.CannotOverwriteExistingCassetteException", which having had a read, seems to be a concurrency issue with the Cassettes.  I am assuming this is likely to be an issue with my environment.  Seems like it will be a bit of newbie issue/question, is here the best place to discuss it?

**Reply by Milind220** [reply]
Hey @Miller2014! We'd love to have you contribute to Ozone! 

Haha oh yeah the error you're seeing is due to the fact that we recently fixed several tests, and the cassettes that the tests use to get mock API responses haven't been re recorded yet. We're going to re record and commit those soon, at which point everything should pass. 

As of now, just to try out the tests, you can try the following:
1. Create a file called .env in the root directory for your Ozone clone. Inside it add the line `WAQI_TOKEN=YOUR_PRIVATE_TOKEN`
2. Run `pytest --disable-recording`. This tells pytest to use a live connection to the WAQI API to conduct the tests, and uses your API token to do this.

Hope you enjoy using and contributing to Ozone! Feel free to ask if you're not sure about anything :)

**Reply by Miller2014** [reply]
Thanks @Milind220. I have done as you suggested, which works. I have one failure which I am looking into.  I will check the current issues and raise an issue if I think it is genuine. 

I look forward to getting more involved and hopefully pick up some tasks/issues.

Joe

**Reply by Milind220** [reply]
@Miller2014 Excellent! Yesterday I fixed one faulty test and re recorded the cassettes, so you could check if that fixed the failure that you're seeing :)

**Reply by Miller2014** [reply]
Thanks @Milind220, yes I see you have fixed the issues. I will start looking at the beginner tasks/issues.
